### PR TITLE
arelle: 2017-08-24 -> 2017-10-07

### PR DESCRIPTION
**Pull request by roberth**
* Redundant \m issue has been solved, simplifying patch
 * Patch now only concerns imports, so renaming
 * The py3to2 tool is not a dependency any more
 * Sadly, the imports are broken again in Python >=3.5.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by roberth** [comment]
Travis fails because nox wants to build `python27Packages.arelle`, `python36Packages.arelle` etc, which are now disabled. I have tested `arelle` (= `python34Packages.arelle`) on NixOS.

File: pkgs/development/python-modules/arelle/default.nix [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
   ... }:
 
 buildPythonPackage rec {
   name = "arelle-${version}${lib.optionalString (!gui) "-headless"}";
-  version = "2017-08-24";
+  version = "2017-10-07";
 
-  disabled = !isPy3k;
+  # Import semantics have changed in python 3.5. Upstream uses python 3.4.
+  disabled = !isPy3k && (!pythonAtLeast "python35");
```
Comment (Mic92) [inline pkgs/development/python-modules/arelle/default.nix]: It stills tries to build for python 3.6 on my system. How about using `isPy34` instead?

Comment (roberth) [reply pkgs/development/python-modules/arelle/default.nix]: Thank you. I should have thought a bit more deeply about all those negations... I'm going for `isPy34` instead. I didn't like `pythonAtLeast` anyway.

**Comment by roberth** [comment]
No, wait, py36 is working again. Either someone fixed a python interpreter problem, or this thing is somehow nondeterministic.

**Pull request closed without merging**